modules/new_serverpod_auth/serverpod_auth_profile/serverpod_auth_profile_server/lib/src/models/user_profile.spy.yaml

```diff
@@ -0,0 +1,36 @@
+class: UserProfile
+serverOnly: true
+table: serverpod_auth_profile_user_profile
+fields:
+  id: UuidValue?, defaultPersist=random
+
+  ### The [AuthUser] this profile belongs to
+  authUser: module:auth_user:AuthUser?, relation(onDelete=Cascade)
+
+  ### The first name of the user or the user's nickname.
+  userName: String?
+
+  ### The full name of the user.
+  fullName: String?
+
+  ### The verified email address of the user.
+  ###
+  ### This should only be set by authentication providers that have
+  ### checked ownership of this email for the user.
+  ###
+  ### Stored in lower-case.
+  email: String?
+
+  ### The time when this user was created.
+  created: DateTime, default=now
+
+  ### The user's profile image
+  image: UserProfileImage?, relation(optional)
+
+indexes:
+  serverpod_auth_profile_email:
+    fields: email
```

Comment:
Currently, email uniqueness is not enforced by the module.

But look-up via email only succeeds if there is exactly 1 profile with that email (thus the developer would have enforced this so far and keeps enforcing it with the lookup).

Reply:
> This prevents a single user from having multiple accounts but displaying the same email. Do we really want that?

Currently this is not enforced by the schema. But the email endpoint would be using this lookup to find an existing account by email in case the sign-up happens as a guest.

I think this is just he user consolidation as I understood we had it.

(It might be debatable whether the consolidation should happen via the profile (which also be for display etc. and can only hold 1 email at a time) or by connecting all the auth providers.)

So the question is, did anything regarding email based user consolidation change, or would you rather just see it done "outside" of the profile?

Reply:
Decision:
- Remove the email look up here, as we have to support cross-authentication provider consolidation anyway (least for all the projects not using the `profile` package)

Reply:
Removed the email based lookup